### PR TITLE
Improve installation process

### PR DESCRIPTION
**Pull request by rhaschke**
See individual commit messages for details

### CODE REVIEW

File: doc/source/installation.rst [hunk 1]
```diff
@@ -107,6 +107,18 @@ Then install the Python packages in the requirements.txt file
...
+
+.. code-block:: shell
+
+    # install packages into catkin workspace instead of ~/.local
+    export PYTHONUSERBASE=~/workspace/ros/devel
+    # don't install packages that are available in system
+    export PIP_IGNORE_INSTALLED=0
+
+    pip3 install -r requirements.txt
+    pip3 install -r src/neem_interface_python/requirements.txt
```
Comment (Tigul) [inline doc/source/installation.rst:121]: Please remove this line, since the neem_interface_python is not part of the repo anymore.

File: doc/source/remarks.rst [hunk 2]
```diff
@@ -33,14 +33,3 @@ Dirty Terminals
 
 If your terminal gets polluted by PyBullet complaining about incomplete URDF descriptions, you need to first fix your
 URDF files by inserting the missing tags and second delete the `resources/cached` folder.
-
-Missing pr2_arm_kinematics
-==========================
```
Comment (Tigul) [inline doc/source/remarks.rst:38]: Why did you remove this part?

File: doc/source/troubleshooting.rst [hunk 3]
```diff
@@ -108,32 +108,3 @@ real_robot environments. This is also explained in the `Action Designator Exampl
 
    with simulated_robot:
       NavigateAction([Pose()]).resolve().perform()
-
-Missing pr2_arm_kinematics
-==========================
```
Comment (Tigul) [inline doc/source/troubleshooting.rst:113]: Same here, why remove this?

**Review by Tigul — CHANGES_REQUESTED** [review]

File: doc/source/installation.rst (hunk 1 above)
Comment (rhaschke) [reply doc/source/installation.rst:121]: Ok. I didn't use that and thus didn't notice.

File: doc/source/remarks.rst (hunk 2 above)
Comment (rhaschke) [reply doc/source/remarks.rst:38]: The arm kinematics error was caused by a missing dependency to `moveit_kinematics`. As I added that dependency in ef50689f64ff05106015a3375b037c4037c0b1c4, the error should be gone now.

Comment (Tigul) [reply doc/source/remarks.rst:38]: Yeah but that error occurred for some people randomly after some time of usage, so I would like to keep this for documentation at least in the troubleshooting section.

Comment (rhaschke) [reply doc/source/remarks.rst:38]: As stated in the removed doc, the package `moveit_kinematics` was auto-removed by apt after some time, which happened because the dependency to `moveit_kinematics` was not declared. As it is now, the package will not be removed accidentally anymore. Trust me ;-)
If you insist, I will revert that part.

Comment (Tigul) [reply doc/source/remarks.rst:38]: Ok, then I will trust you on that :)
